autoload/lightline/colorscheme/papercolor_dark.vim

```diff
@@ -0,0 +1,257 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/autoload/lightline/colorscheme/papercolor_light.vim
+" Author: TKNGUE
+" License: MIT License
+" Last Change: 2015/07/21 13:18:03
+" =============================================================================
+
+fun! <SID>grey_number(x) "{{{
```

Comment:
- Do not use abbreviation; use `function! ... endfunction`.
- Do not use `<SID>` but to use `s:`.